strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/test/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticationProviderTest.java

```diff
@@ -51,6 +57,17 @@
     @Inject
     private LdapUserDetailsService ldapUserDetailsService;
 
+    @Inject
+    private PasswordEncoder passwordEncoder;
+
+    private static CustomLdapUserDetailsMapper userDetailsMapper;
+
+
+    @BeforeAll
+    private static void init(){
+        userDetailsMapper = new CustomLdapUserDetailsMapper();
+    }
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
    private static void init()
    {
        userDetailsMapper = new CustomLdapUserDetailsMapper();
    }
```